Formula/llvm.rb

```diff
@@ -1,8 +1,8 @@
 class Llvm < Formula
   desc "Next-gen compiler infrastructure"
   homepage "https://llvm.org/"
-  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/llvm-project-12.0.0.src.tar.xz"
-  sha256 "9ed1688943a4402d7c904cc4515798cdb20080066efa010fe7e1f2551b423628"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.1/llvm-project-12.0.1.src.tar.xz"
+  sha256 "129cb25cd13677aad951ce5c2deb0fe4afc1e9d98950f53b51bdcfb5a73afa0e"
   # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
   license "Apache-2.0" => { with: "LLVM-exception" }
   revision 1
```

Comment:
```suggestion
```

Reply:
Note: You may find that using `brew bump-formula-pr` makes this very simple.

Reply:
why is this needed?

Reply:
https://docs.brew.sh/Formula-Cookbook#formulae-revisions

These are used when there are updates to a formula without a version update, or a recompile that users must upgrade to. For new versions, `revision` must be 0 (which means you can remove this line).